Formula/qt.rb

```diff
@@ -56,9 +66,16 @@ def install
       -no-rpath
       -pkg-config
       -dbus-runtime
-      -proprietary-codecs
     ]
 
+    if Hardware::CPU.arch == :arm64
+      # Temporarily fixes for Apple Silicon
+      args << "-skip" << "qtwebengine" << "-no-assimp" << "QMAKE_APPLE_DEVICE_ARCHS=arm64"
```

Comment:
May want to note the `qtwebengine` omission in `caveats`?

Reply:
Yeah, if we get it working, I'll add some caveats